lib/cdo/delete_accounts_helper.rb

```diff
@@ -111,6 +111,7 @@ def clean_and_destroy_pd_content(user_id)
     Pd::RegionalPartnerProgramRegistration.where(user_id: user_id).update_all(form_data: '{}', teachercon: 0)
     Pd::Teachercon1819Registration.where(user_id: user_id).update_all(form_data: '{}', user_id: nil)
     Pd::RegionalPartnerContact.where(user_id: user_id).update_all(form_data: '{}')
+    Pd::RegionalPartnerMiniContact.where(user_id: user_id).update_all(form_data: '{}')
```

Comment:
There's no test in `delete_accounts_helper_test` that tests this behavior for regional partner contacts. Thinking maybe I should add, in case someone deleted the model one day (so we'd catch a regression to the account deletion process)?

I did want to delete the `pd_regional_partner_contact` factories though in [part 3 of this PR series](https://github.com/code-dot-org/code-dot-org/pull/33051), which would be useful to keep around if I'm going to add tests here. Any thoughts on how to proceed? I could just tear the factory down to the bare minimum to be usable for this test?

Reply:
I think it makes sense to add a test and just keep around the minimal factory required for it.

Reply:
I was wrong (should have known there was a test given I think @islemaster wrote this 😄 ) -- there is a test for deleting data from regional partner contacts, which I just duplicated for regional partner mini contacts. I made the regional partner contact factory the minimum to keep this test around.